src/net_processing.cpp

```diff
@@ -1978,7 +1978,7 @@ void PeerManagerImpl::ProcessGetBlockData(CNode& pfrom, Peer& peer, const CInv&
                 if (a_recent_compact_block && a_recent_compact_block->header.GetHash() == pindex->GetBlockHash()) {
                     m_connman.PushMessage(&pfrom, msgMaker.Make(NetMsgType::CMPCTBLOCK, *a_recent_compact_block));
                 } else {
-                    CBlockHeaderAndShortTxIDs cmpctblock(*pblock, true);
+                    CBlockHeaderAndShortTxIDs cmpctblock(*pblock);
```

Comment:
```suggestion
                    CBlockHeaderAndShortTxIDs cmpctblock{*pblock};
```

nit: `{}`

Reply:
Done (here and all other lines I touched)